src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,23 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.2
+
+This is a bugfix release:
+
+* The Globalnet metrics port will now be opened by default when Globalnet is deployed using `subctl cloud prepare`.
+* Submariner ServiceExport now has unique condition types to simplify waiting for readiness.
+* The `subctl diagnose` command now supports NAT-discovery port validation.
+* The `subctl cloud prepare rhos` command will now work properly for nodes to which security groups were added manually.
+* The `submariner-operator` namespace is labeled in accordance with _KEP-2579: Pod Security Admission Control_ (default in Kubernetes 1.24)
+to allow the privileged Pods required for Submariner's interactions with iptables.
```

Comment:
```suggestion
to allow the privileged Submariner Pods.
```

IPtables interaction is just one of the things we do on the nodes. Apart from iptables, we also configure IPsec tunnels, routes, procs entries etc. So lets avoid mentioning iptables and just say that its for privileged submariner pods.

Reply:
How about this wording?

```suggestion
to allow the Pods to be privileged.
```